packages/components/src/components/progress-bar/progress-bar.css

```diff
@@ -10,20 +10,25 @@
  */
 
 :host {
-  --background: var(--telekom-color-primary-standard);
-  --background-disabled: var(--telekom-color-ui-disabled);
-  --background-color: var(--telekom-color-ui-faint);
-  --color-error: var(--telekom-color-text-and-icon-functional-danger);
-  --background-error: var(--telekom-color-functional-danger-subtle);
-  --color-success: var(--telekom-color-text-and-icon-functional-success);
+  --track-color: var(--telekom-color-ui-faint);
```

Comment:
better naming now but will this be a problem/breaking change for people who are using the old variables?

Reply:
yes, this is "breaking" 🫣 — we can bare a little… 🙏